tests/scripts/test_ingest.py

```diff
@@ -152,6 +152,41 @@ def test_ingest_append(raster_file, tmpworkdir):
     assert all((ds,) in driver.get_datasets() for ds in ('img1', 'img2'))
 
 
+@pytest.mark.parametrize('addflags', [['--skip-existing', ], []])
+def test_ingest_append_overwrite(addflags, raster_file, tmpworkdir):
+    from terracotta.scripts import cli
+
+    same_name = 'myimage'
+    for infile in (f'dir1/{same_name}.tif', f'dir2/{same_name}.tif'):
+        temp_infile = tmpworkdir / infile
+        os.makedirs(temp_infile.dirpath(), exist_ok=True)
+        shutil.copy(raster_file, temp_infile)
+
+    outfile = tmpworkdir / 'out.sqlite'
+
+    runner = CliRunner()
+    result = runner.invoke(cli.cli, ['ingest', 'dir1/{name}.tif', '-o', str(outfile)] + addflags)
+    assert result.exit_code == 0
+    assert outfile.check()
+
+    runner = CliRunner()
+    result = runner.invoke(cli.cli, ['ingest', 'dir1/{name}.tif', '-o', str(outfile)] + addflags)
+    assert result.exit_code == 0
+    assert outfile.check()
+
+    result = runner.invoke(cli.cli, ['ingest', 'dir2/{name}.tif', '-o', str(outfile)] + addflags)
+    assert result.exit_code == 0
+    assert outfile.check()
+
+    from terracotta import get_driver
+    driver = get_driver(str(outfile), provider='sqlite')
+    assert driver.key_names == ('name',)
+    keys = list(driver.get_datasets())
+    assert len(keys) == 1
+    assert len(keys[0]) == 1
+    assert keys[0] == (same_name, )
+
```

Comment:
This should test that the raster points to the correct path, too

Reply:
Good point.

Reply:
Done.